internals/cli/cmd_help.go

```diff
@@ -196,37 +199,35 @@ var HelpCategories = []HelpCategory{{
 	Commands:    []string{"warnings", "okay"},
 }}
 
-var (
-	longPebbleDescription = strings.TrimSpace(`
-Pebble lets you control services and perform management actions on
-the system that is running them.
-`)
-	pebbleUsage               = "Usage: pebble <command> [<options>...]"
-	pebbleHelpCategoriesIntro = "Commands can be classified as follows:"
-	pebbleHelpAllFooter       = "Set the PEBBLE environment variable to override the configuration directory \n" +
-		"(which defaults to " + defaultPebbleDir + "). Set PEBBLE_SOCKET to override \n" +
-		"the unix socket used for the API (defaults to $PEBBLE/.pebble.socket).\n" +
-		"\n" +
-		"For more information about a command, run 'pebble help <command>'."
-	pebbleHelpFooter = "For a short summary of all commands, run 'pebble help --all'."
-)
+func longPebbleDescription() string {
+	return fmt.Sprintf(strings.TrimSpace(`
+%s lets you control services and perform management actions on the
+system that is running them
+	`), cmd.Personality.DisplayName)
+}
 
 func printHelpHeader() {
-	fmt.Fprintln(Stdout, longPebbleDescription)
+	fmt.Fprintln(Stdout, longPebbleDescription())
```

Comment:
If you're referring to calling this `printLongPebbleDescription` and have the result printed instead of returned, we do need the return value of this function in `cli.go`.

Reply:
The suggestion was to do less: not change the variable name but tune the text, and use the same function we use elsewhere for performing the variable replacement. Am I missing something?

Reply:
I did miss the point of your feedback (: I will instead keep the same variables and call `cli.applyPersonality()` throughout the code, is that what you mean? Thanks!

Reply:
Yeah, thanks :+1: